src/components/NewsItem/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import testImage from './Temple_Island.jpg';
+
+function NewsItem(props) {
+  return (
+    <div className={cx('NewsItem', {'NewsItem--featured':props.featured})}>
+      <div className="TileImage">
+        <img className='Image' src={testImage} />
+      </div>
+      <div className="TileDetails">
+        <div className = "Title"><h2 className="h-title">{props.title}</h2></div>
```

Comment:
If you run `yarn run format` before commuting, it fill fix all the formatting tweaks automatically